packages/kbn-securitysolution-autocomplete/src/translations/index.ts

```diff
@@ -27,3 +27,6 @@ export const NUMBER_ERR = i18n.translate('autocomplete.invalidNumberError', {
 export const DATE_ERR = i18n.translate('autocomplete.invalidDateError', {
   defaultMessage: 'Not a valid date',
 });
+export const FIELD_SPACE_WARNING = i18n.translate('autocomplete.fieldSpaceWarning', {
+  defaultMessage: 'Warning: there is whitespace that isn’t being displayed',
```

Comment:
Starting with one of @nastasha-solomon's [options](https://github.com/elastic/kibana/pull/139617#issuecomment-1229970670), with a few extra edits. I think "spaces" is more appropriate in this context than "whitespace"; the latter is more of a visual graphic design concept, but this warning is really just about text-based space characters.
```suggestion
  defaultMessage: 'Warning: There are spaces at the start or end of this value that aren’t being displayed.',
```

Reply:
Here's an even shorter version. I think it works for the exceptions editor and viewer:
```suggestion
  defaultMessage: 'Warning: Spaces at the start or end of this value aren't being displayed.',
```

Reply:
Awesome thanks @nastasha-solomon @joepeeples I'll update it ^_^